tests/utils/test_sm_convert.py

```
@@ -84,8 +86,9 @@ def test_post_init(ROSETTA_PYTHON_SCRIPTS, ROSETTA, ROSETTA3, PYTHON_SCRIPTS_PAT
     os.environ["ROSETTA"] = ROSETTA
     os.environ["ROSETTA3"] = ROSETTA3
 
-    generator = SmallMoleculeParamsGenerator(num_conformer=50, save_dir="./test_ligands/")
-    assert os.path.abspath(generator._rosetta_python_script_dir) == os.path.abspath(PYTHON_SCRIPTS_PATH)
+    with tmpdir_manager() as test_ligands:
```

Ruff

Local variable `test_ligands` is assigned to but never used

Remove assignment to unused variable `test_ligands`

(F841)